crates/benches/contracts/src/models/character.cairo

```diff
@@ -1,7 +1,8 @@
 use starknet::{ContractAddress, get_caller_address};
 
 // TODO import all this when complex benchmarks are merged
-#[derive(Model, Copy, Drop, Serde)]
+#[derive(Introspect, Copy, Drop, Serde)]
+#[dojo::model]
```

Comment:
In my original PR I removed auto deriving Introspect for models but I think we should probably add it back. Since there I don't think there is a case where we want model without introspect. Not necessary to land in this PR though